package.json

```diff
@@ -11,13 +11,14 @@
     "dev": "vuepress dev",
     "check-links": "yarn check-md && yarn check-config-paths",
     "check-md": "vuepress check-md .",
-    "check-style": "yarn lint && yarn prettier-md-check",
+    "check-style": "yarn style",
```

Comment:
Why does not use "style" directly? (We need to change the README and the CI)

Or simply removing `style` rule and just having:
```
"check-style": "yarn lint && yarn marklint"
```

Reply:
Yes this is only to match the syntax of `check-md`. 
I thought about removing it but I was not sure. If you don't like it either I remove it